packages/apps/plugins/plugin-thread/src/components/types.ts

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import type { Thread as ThreadType } from '@braneframe/types';
+import { type Space } from '@dxos/react-client/echo';
+import type { ThreadProps } from '@dxos/react-ui-thread';
+
+export type ThreadContainerProps = {
+  space: Space;
+  thread: ThreadType;
+  currentRelatedId?: string;
+  autoFocusTextBox?: boolean;
```

Comment:
just autoFocus

Reply:
I disagree, where a property is a React- or HTML-native attribute, that is understood to apply to the outer element, which in this case would be `Thread`, not `MessageTextbox` which is a child. Any message could be autofocused, and it may eventually somehow be possible to autofocus the thread itself, so I don’t think the ambiguity is worthwhile.

We already qualify props when the name alone is not sufficient, e.g. `onCreateInvitationClick` instead of just `onClick` (since multiple children could be clicked).